vendor/github.com/intel/govmm/qemu/qmp.go

```diff
@@ -946,6 +946,23 @@ func (q *QMP) ExecuteBlockdevDel(ctx context.Context, blockdevID string) error {
 	return q.executeCommand(ctx, "x-blockdev-del", args, nil)
 }
 
+func (q *QMP) ExecuteChardevDel(ctx context.Context, chardevID string) error {
```

Comment:
Files below `vendor/` should not be modified directly as the must be pristine upstream files. See https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#re-vendor-prs.